pkg/dockerregistry/server/wrapped/zz_wrapper.go

```diff
@@ -0,0 +1,7 @@
+package wrapped
```

Comment:
why "zz_wrapper.go" and not just "wrapper.go"?

Reply:
renamed